src/Administration/Resources/app/administration/src/module/sw-sso-error/snippet/de-DE.json

```diff
@@ -1,15 +1,15 @@
 {
   "sw-sso-error": {
     "general": {
-      "title": "SSO Fehler",
+      "title": "SSO-Fehler",
       "description": "Shopware-Login-Fehler"
     },
     "error-card": {
       "title": "Kein Zugriff auf diesen Shop",
       "text": "Deine Anmeldung war erfolgreich, aber du hast keinen Zugriff auf diesen Shopware Shop.",
```

Comment:
```suggestion
      "text": "Deine Anmeldung war erfolgreich, aber du hast keinen Zugriff auf diesen Shopware-Shop.",
```

Reply:
Not necessary, since those are 2 english terms